CODE_OF_CONDUCT.md

```diff
@@ -23,7 +23,7 @@ affect a person's ability to participate within them.
 
 If you believe someone is violating the code of conduct, we ask that you report
 it by emailing <conduct@riot-rs.org>. For more details please see our [Reporting
-Guidelines](https://future-proof-iot.github.io/RIOT-rs/dev/CoC_reporting.html).
+Guidelines](https://github.com/future-proof-iot/RIOT-rs/blob/main/book/src/COC_reporting.md).
```

Comment:
https://future-proof-iot.github.io/RIOT-rs/dev/COC_reporting.html actually works, I think just the O in COC is off?

Reply:
Ah indeed! I will change it back to the original link, with the typo 'O' fixed.

Might take some days, since I'm still learning the git workflow.